grakn-engine/src/main/java/ai/grakn/engine/user/SystemKeyspaceUsers.java

```diff
@@ -126,9 +127,9 @@ public Json getUser(String username) {
         if (superUsername().equals(username)) {
             return Json.object(USER_NAME, username);
         }
-        try (GraknGraph graph = factory.getGraph(SystemKeyspace.SYSTEM_GRAPH_NAME, GraknTxType.READ)) {
-            VarPattern lookup = var("entity").isa(USER_ENTITY).has(USER_NAME, username);
-            VarPattern resource = var("property");
+        try (GraknGraph graph = factory.getGraph(SystemKeyspace.SYSTEM_GRAPH_NAME, GraknTxType.READ)) {VarPattern lookup = var("entity").isa(USER_ENTITY).has(USER_NAME, username);
+        Var resource = var("property");
```

Comment:
Yes. `resource` is just a variable name - it is only used to identify concepts in a query. Whereas, `lookup` is a pattern applied to the variable `var("entity")`.

Making these kind of distinctions is the main reason I made this change. I'll send you some more details on the motivation if you like.